client/src/Components/AddVideo.js

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+
+const AddVideo = ({ videos, setVideos }) => {
+  const [videoName, setVideoName] = useState("");
+  const [videoLink, setVideoLink] = useState("");
+  function handleInputChange(event) {
+    if (event.target.name === "name") {
+      setVideoName(event.target.value);
+    } else if (event.target.name === "url") {
+      setVideoLink(event.target.value);
+    }
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    let video = {
+      title: videoName,
+      url: videoLink,
+    };
+    if (video.url.startsWith("https://www.youtube.com/")) {
+      fetch(`https://paulina-full-stack-project-server.onrender.com/videos`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(video),
+      })
+        .then((res) => res.json())
+        .then((data) => {
+          setVideos(data);
+        })
+        .catch((error) => console.log(error));
+    } else {
+      alert("Please ensure fields are filled in correctly");
```

Comment:
In general if you're telling a user they've done something wrong, telling them what's expected is useful - I'd maybe change this to something like:
```suggestion
      alert(`Expected video URL to start with 'https://www.youtube.com/' but you provided '${video.url}' - please ensure fields are filled in correctly`);
```